tests/AppBundle/Controller/Admin/BlogControllerTest.php

```diff
@@ -35,31 +35,31 @@ class BlogControllerTest extends WebTestCase
     /**
      * @dataProvider getUrlsForRegularUsers
      */
-    public function testRegularUsers($httpMethod, $url, $statusCode)
+    public function testAccessDeniedForRegularUsers($httpMethod, $url)
     {
-        $client = static::createClient([], [
+        $client = self::createClient([], [
             'PHP_AUTH_USER' => 'john_user',
             'PHP_AUTH_PW' => 'kitten',
         ]);
 
         $client->request($httpMethod, $url);
-        $this->assertSame($statusCode, $client->getResponse()->getStatusCode());
+        $this->assertTrue($client->getResponse()->isForbidden());
```

Comment:
In this case, `isForbidden()` only checks the `403` status code ... so the code it's technically equivalent...